docs/versioned_docs/version-1.x/cli-commands.md

```diff
@@ -852,11 +852,11 @@ const QuotePage = ({ id }) => {
     <>
       <h1>QuotePage</h1>
       <p>Find me in "./web/src/pages/QuotePage/QuotePage.js"</p>
-      <p>
```

Comment:
I don't think we should update the docs for older versions (like v1 in this case). Because if someone actually was on that older version they'd still get the old behavior. Only updating files outside the `versioned_docs` directory is all that's needed 🙂